MANIFEST.in

```diff
@@ -1,22 +1,20 @@
 graft src
-graft tests
-graft docs
 
-include .cookiecutterrc
-include .editorconfig
+prune config
+prune docs
+prune requirements
+prune scripts
+prune tests
 
+exclude * .*
```

Comment:
This seems to be too intrusive? Which files do you intend to exclude? I would definitely keep `docs`, `config` and `requirements`. `tests` should be excluded by the `setup.py` itself.

Reply:
Honestly, I intend to exclude as much as I can. 

- `.cookiecutterrc`:  development related, not needed
- `.coverage`: test-execution related, not needed
- `.editorconfig`: development related, not needed
- `pyproject.toml`: only used for black right now, so test-related, not needed if we don't include tests
- `runtests.py`: test-related, not needed if we don't include tests
- `tox.ini`: dev/test-related, not needed if we don't include tests
- `.travis.yml`: ci-related, not needed

I searched and searched over the internet but I can't find any post about Python packaging emphasizing the need to include development-related files in sdists/wheels. And I personally don't see what are the benefits would be.

- `tests/`: I never ran tests against installed packages, and I never heard of anyone doing it. Also, why should they be excluded by `setup.py` and not by `MANIFEST.in`? I find it more consistent to put that rule in the manifest.
- `docs/`: I never read the docs of an installed packages (from the local, installed files), and I never heard of anyone doing it, but for this one I agree we could add it (the built version, not the raw RST files), because it still could be useful when in offline mode.
- `config/`: we don't have this directory anymore so I should remove that line from MANIFEST.in :slightly_smiling_face: 
- `requirements/`: only useful for development/CI purposes as it's used by tox only. All the "production" requirements are listed in `setup.py`.

Of course these are personal opinions, so please prove me wrong with examples :slightly_smiling_face:. Or maybe I'm missing something obvious :confused:

Reply:
By default, source distribution only includes files handled by `setup.py`, so `MANIFEST.in` should only list files you want to include. As a result, you don't need to exclude files you don't want.

The `MANIFEST.in` on master looks good enough to me. If you don't want some of the files, you should be fine with just removing those lines from the manifest. Tests doesn't seem to be included by `setup.py`, so they doesn't need to be mentioned in manifest neither, they will not be installed.

I'm not quite sure about details of source version, but in my opinion they should include all files needed to build the application and its docs (I don't use them either, but some admins complain about missing docs). There is somewhat trouble with tests as they should be run as part of the build process, e.g. building a Debian package, but I'm not quite sure, how it should affect the source distribution.

Reply:
I think I explicitly excluded files, even if it was not necessary, because the old `python setup.py check` command would complain about versioned files not included by the manifest. Maybe it's different now with `twine check`. Will see that.

Reply:
I doesn't seem to be a problem any more. Let me know if you want me to check another PR.